package.json

```diff
@@ -33,6 +33,9 @@
   },
   "dependencies": {
     "adiff": "^0.2.13",
+    "react": "^15.3.1",
```

Comment:
Should we switch to using the installed package instead of using it as an external?